File: prob003/python/prob003.py
import sys
f = open(sys.argv[1],'r')#sys call

#reference:http://stackoverflow.com/questions/16233593/how-to-strip-comma-in-python-string
#remove new line,comma and period. rstrip does not work for commas.
line = f.readline().rstrip('\n.').replace(',','')

wordList = line.split(' ')
for word in wordList:
   print len(word)

f.close()

File: prob003/ruby/prob003.rb
#http://stackoverflow.com/questions/7157207/how-can-i-read-a-file-with-ruby
contents = File.read(ARGV[0])

#http://stackoverflow.com/questions/17348539/how-to-strip-commas-from-float-input
contents = contents.gsub(',','').gsub('.','')

#http://stackoverflow.com/questions/13537920/ruby-split-by-whitespace
contents = contents.split(' ')

contents.each do |word|
   puts word.length
end

#http://www.rubylife.jp/ini/for/index5.html
#puts contents

